### ISSUE TITLE
About the concatenation between the audio and visu

### ISSUE THREAD

**Issue by RifqiAW** [mpc001/end-to-end-lipreading#20]
Hi, thank you for the code. I want to ask about the concatenation part,
inputs = torch.cat((audio_outputs, video_outputs), dim=2)
        outputs = concat_model(inputs)
Does this mean it is being concatenated along the feature axis? Is it because the audio and visual input has different number of timesteps?

**Comment by mpc001** [comment]
Hi, 

Audio features and video features are concatenated along the feature axis. We downsample the audio input to 25 frames per second by convolutional layers and pooling layers.

**Comment by gopzgopez** [comment]
Hi,

Did you also use convolutional layers when you tried to use MFCC instead of ResNet?
